proposals/NNNN-atf-improvment-proposal.md

```diff
@@ -0,0 +1,79 @@
+# Improve Automated test framework
+
+* Proposal: [SDL-NNNN](NNNN-atf-improvement-proposal.md)
+* Author: [Alexander Kutsan](https://github.com/smartdevicelink)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / ATF]
+
+## Introduction
+
+The proposal is to make ATF more flexible, salable and easy to use and integrate with popular CI systems and to analyze and full of all required information. 
```

Comment:
The proposal is to make ATF more flexible, scalable and easy to use and integrate with popular CI systems. ATF should be full of all required information and easy to analyze.